assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -61,6 +70,7 @@ const EditorWizardModal = () => {
 	}, [] );
 
 	useEffect( () => {
+		showSenseiTour( false );
```

Comment:
Yap we can do that, but in our case, we're assuming (till now) that if a tour is loaded, it's meant to be shown, unless some component wants it otherwise. For example, we show this Tour even when the modal is not shown (in the scenario, when an existing Course is opened for editing, not creating a new course). So as the editor wizard modal will not be there in that case, so we'll need to set it to True from some other place. So that's why I've kept it like it is now. But these are not very strong reasons and I don't have a strong preference. We can change it later too if different use cases come up.

Reply:
Okay! ;)